test-packages/my-embroider-app-test.mjs

```diff
@@ -22,6 +22,7 @@ describe('app coverage generation', function () {
         "@embroider/webpack": embroiderVersion
       });
   
+      await execa('rm',  ['-rf', '.embroider'], { cwd: buildPath, env });
```

Comment:
Can you please explain why this was needed?

Reply:
@RobbieTheWagner  so currently we are testing against `emroider/core` versions from 0 to 3.
During each build it creates a `.embroider` folder, but seems like when we run a test for a newer version it will try to re-use the `.embrodier` folder generated by the previous test and it was leading to some unpredictability.

For example there is a problem with [babel plugin serialization](https://github.com/embroider-build/embroider/issues/1525) in v3 that I solved [here](https://github.com/ember-cli-code-coverage/ember-cli-code-coverage/pull/405/files#diff-220612c243d58fc738e2512fd4efbd91e7700e2b8b547032a3e260df5fb7ee9dR92) using [this](https://github.com/Serabe/ember-bind-helper/pull/21) as a reference, but it would still fail on tests w/o cleaning up `.embroider` because it was relying on what was generated by the previous embroider version

So basically all I did was make sure the tests are totally isolated from each other and don't affect each other

Reply:
@vstefanovic97 that's good knowledge! Mind to drop a comment in the code with this explanation for future readers?

Reply:
Added a comment in the code about this @SergeAstapov, @RobbieTheWagner